test.md

```diff
@@ -0,0 +1,6 @@
+<!-- this should be ignored by git -->
+[]: # 
+[]: # ## 📚 License
+[]: # 
+[]: # This project is licensed under the MIT License.
+[]: #
```

Comment:
To maintain our git history clean, let's make sure to add `test.md` file to the [.gitignore](https://stackoverflow.com/questions/13675216/why-should-i-use-gitignore) file. What do you think? The purpose of gitignore files is to ensure that certain files not tracked by git remain untracked.